miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/MiniApp.kt

```diff
@@ -49,6 +49,25 @@ abstract class MiniApp internal constructor() {
         miniAppNavigator: MiniAppNavigator? = null
     ): MiniAppDisplay
 
+    /**
+     * Creates a mini app.
+     * The mini app is downloaded, saved and provides a [MiniAppDisplay] when successful.
```

Comment:
I think this should mention something like this:
```suggestion
     * Creates a mini app using the mini app ID and version specified in [MiniAppInfo].
     * This should only be used in "Preview Mode".
     * The mini app is downloaded, saved and provides a [MiniAppDisplay] when successful.
```